### PR TITLE
Update both platforms version to 1.0.0

### PR DESCRIPTION
**Pull request by paultiarks**

### CODE REVIEW

**Comment by bhariharan** [comment]
@paultiarks Can we make this `Swift 5.0` before tagging it `1.0.0`?

**Comment by paultiarks** [comment]
@bhariharan Unless I'm mistaken, this is already be Swift 5 compatible.

**Comment by bhariharan** [comment]
Awesome!

File: platforms/android/settings.gradle [hunk 1]
```diff
@@ -8,4 +8,4 @@
  */
 
 include ':demoapp', ':barcodescanner'
-rootProject.name='Nimubs Barcode Scanner Plugin'
+rootProject.name='Nimbus Barcode Scanner Plugin'
```
Comment (jamie-houston) [inline platforms/android/settings.gradle:11]: 😱

**Pull request merged**